sourcegraph/sourcegraph.proto

```diff
@@ -355,6 +355,8 @@ message ReposListBranchesOp {
 }
 
 message RepoListBranchesOptions {
+	bool include_commit = 4;
```

Comment:
It's not a big deal, but were there ever fields with tags 1 and 2? I don't know why the numbering started at 3.